backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/appsearchmodal/internal/BpkSection.kt

```diff
@@ -65,11 +65,14 @@ internal fun BpkSectionHeading(
 }
 
 @Composable
-internal fun BpkSectionItem(item: BpkItem, modifier: Modifier = Modifier, behaviouralCallback: BehaviouralCallback? = null) {
+internal fun BpkSectionItem(item: BpkItem, modifier: Modifier = Modifier, clickHandleScope: BpkClickHandleScope? = null) {
     Row(
         modifier = modifier
             .fillMaxWidth()
-            .registerForBehaviouralEvents(item = item, behaviouralCallback = behaviouralCallback, onClick = item.onItemSelected),
+            .clickable {
```

Comment:
this also means we can reuse this for other types of events (ie making something selectable, rather than clickable to ensure the right accessibility announcements are made)